packages/dht/src/dht/DhtNode.ts

```diff
@@ -202,14 +209,17 @@ export class DhtNode extends EventEmitter<Events> implements ITransport {
                 maxConnections: this.config.maxConnections
             }
             // If own PeerDescriptor is given in config, create a ConnectionManager with ws server
-            if (this.config.peerDescriptor && this.config.peerDescriptor.websocket) {
+            if (this.config.peerDescriptor && this.config.peerDescriptor.websocket && isNodeJS()) {
                 connectionManagerConfig.webSocketHost = this.config.peerDescriptor.websocket.ip
-                connectionManagerConfig.webSocketPort = this.config.peerDescriptor.websocket.port
+                connectionManagerConfig.webSocketPortRange = { 
+                    min: this.config.peerDescriptor.websocket.port,
+                    max: this.config.peerDescriptor.websocket.port
+                }
             } else {
                 // If webSocketPort is given, create ws server using it, webSocketHost can be undefined
-                if (this.config.webSocketPort) {
+                if (this.config.webSocketPortRange) {
                     connectionManagerConfig.webSocketHost = this.config.webSocketHost
-                    connectionManagerConfig.webSocketPort = this.config.webSocketPort
+                    connectionManagerConfig.webSocketPortRange = this.config.webSocketPortRange
                 }
```

Comment:
I'm a bit confused by this else-block.

1. Shouldn't `isNodeJS()` also be checked for truthiness here?

2. What is `this.config.webSocketHost` and how does it relate to `this.config.peerDescriptor.websocket.ip`?

Reply:
Making some clean ups related to this, but the NodeJS does not need to be explicitly checked here as the WS related configs are nullified at line 185.